data/core/encyclopedia/calendar.cfg

```diff
@@ -22,7 +23,7 @@ Summit Star
 Kindlefire
 Stillseed
 Reaper’s Moon
-Verglas Bloom
+Vergla's Bloom
```

Comment:
Needs a typographic-style quotation mark here.

@nemaara is this intended to be a possessive, or is "Verglas" meant to be a word?

Reply:
> Needs a typographic-style quotation mark here.

Done

Reply:
Verglas is a word. Keep it as written.

Reply:
Okay, I should have checked a dictionary before writing my previous comment. However, words that are far less obscure than this one are typically given translation hints.

Reply:
done